sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/complexTypeCreator.scala

```diff
@@ -132,50 +153,80 @@ case class CreateMap(children: Seq[Expression]) extends Expression {
     new ArrayBasedMapData(new GenericArrayData(keyArray), new GenericArrayData(valueArray))
   }
 
+  private def getAccessors(ctx: CodegenContext, dt: DataType, array: String,
```

Comment:
nit: fix the style to be
```
def xxx(
  param1: xxx,
  param2:xxx)
```

and the method name `getAccessors` looks weird, can you add some comments to explain it and its return type?